pkg/cluster/internal/kubeadm/config.go

```diff
@@ -110,6 +110,11 @@ controllerManagerExtraArgs:
   enable-hostpath-provisioner: "true"
 nodeRegistration:
   criSocket: "/run/containerd/containerd.sock"
+  kubeletExtraArgs:
+    fail-swap-on: "false"
+    {{ if .APIAdvertiseAddress -}}
+    node-ip: "{{ .APIAdvertiseAddress }}"
```

Comment:
Ipv6 needs it, otherwise Kubelet picks the ipv4

Reply:
understood.
would the kubelet long term always default to a ipv4 from the default interface?

Reply:
IIRC the logic implemented in Kubernetes always defaults to the IPv4 address ... don´t think that´s a behavior that can change soon ... will add it to my backlog, it can be something interesting to implement though